Source/Scene/DiscardMissingTileImagePolicy.js

```diff
@@ -89,7 +89,9 @@ define([
             that._isReady = true;
         }
 
-        when(resource.fetchImage(true), success, failure);
+        when(resource.fetchImage({
```

Comment:
I know this was existing code, but it's bad practice to use the `(promise, success, failure)` form of when.  Exceptions thrown in `promise` may not get properly handled.  Instead use `promise.then(success).otherwise(failure)` instead.